src/elements/hx-popover/index.spec.js

```diff
@@ -0,0 +1,92 @@
+import { fixture, expect } from '@open-wc/testing';
```

Comment:
Please update this comment in `src/elements/hx-popover/index.js`:

```js
    /**
     * This position attribute is for the popover arrow located on the Shadow DOM element root.
     *
     * @override
     * @param {NormalizedPositionString}
     */
```

https://github.com/HelixDesignSystem/helix-ui/blob/master/src/elements/hx-popover/index.js#L39-L42

Reply:
Changes have been done, please review.